spec/helpers/standard-colors.js

```diff
@@ -0,0 +1,8 @@
+/* exported standardColors */
+
+/* Use these in tests to avoid breaking tests if default colors are changed */
+var standardColors = [
+    '#e41a1c', '#377eb8', '#4daf4a',
+    '#984ea3', '#ff7f00', '#ffff33',
+    '#a65628', '#f781bf', '#999999'
+];
```

Comment:
Why copy `d3.schemeSet1` here? It's not going to change in d3 and if it did, it would still break all our tests.

Reply:
Actually I like the idea of `defaultColors` in config, once we move there, we can use that to match, this variable would not be needed at all.